eta/models/manifest.json

```diff
@@ -78,7 +58,7 @@
                     "labels_path": "{{eta}}/tensorflow/models/research/object_detection/data/mscoco_label_map.pbtxt"
                 }
             },
-            "date_created": "2019-04-11 12:56:53"
+            "date_created": "2019-12-17 08:25:51"
```

Comment:
Do the `google_drive_id` attributes need to be updated for all the models you reimported?

Reply:
The IDs don't need to be updated because I uploaded the newly exported graphs in-place of the old ones